scripts/intersections.py

```diff
@@ -0,0 +1,73 @@
+import configparser
+import os
+import warnings
+import geopandas as gpd
+import pandas as pd
+pd.options.mode.chained_assignment = None
+warnings.filterwarnings('ignore')
+
+CONFIG = configparser.ConfigParser()
+CONFIG.read(os.path.join(os.path.dirname(__file__), 'script_config.ini'))
+BASE_PATH = CONFIG['file_locations']['base_path']
+
+DATA_RAW = os.path.join(BASE_PATH, 'raw')
+DATA_PROCESSED = os.path.join(BASE_PATH, 'processed')
+DATA_RESULTS = os.path.join(BASE_PATH, 'results')
+
+path = os.path.join(DATA_RAW, 'countries.csv')
+
+# Population and floods layer
+population_folder = os.path.join(BASE_PATH, 'processed', 'KEN', 'population', 'shapefiles')
+flood_folder = os.path.join(BASE_PATH, 'processed', 'KEN', 'hazards', 'inunriver', 'shapefiles')
+
+# Output folder
+folder_out = os.path.join(DATA_RESULTS, 'KEN', 'pop_hazard')
+
+def intersect_layers(folder_1, folder_2):
+
+    for firstfiles in os.listdir(folder_1):
```

Comment:
Don't make your iterator plural, or it reads a bit illogically for most python users. It should basically be "for filename in filenames:" or whatever your equivalent is, as the iterator is a single file, whereas the larger data structure contains multiple data items. Beginners are usually taught this way (e.g., for my_iterator in my_data_structure:)